src/adapter/Cargo.toml

```diff
@@ -29,6 +29,9 @@ features = ["postgres", "postgres_backend", "uuid"]
 [dependencies.rust_core]
 path = "../core"
 
```

Comment:
Either place this on top of the file, or at the end

Reply:
I'll put it on the top of dependencies section